conftest.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 
 # This file lists files which should be ignored by pytest
-collect_ignore = ["setup.py", "sopel.py"]
+collect_ignore = []
```

Comment:
Can we just remove this, since it's empty, or will it break something?

Reply:
We could probably remove it, yes.